### PR TITLE
in_winevtlog: Add descriptions about xml query for filtering events

### PR DESCRIPTION
**Pull request by cosmo0920**
This PR is corresponding to https://github.com/fluent/fluent-bit/pull/7848.

### CODE REVIEW

File: pipeline/inputs/windows-event-log-winevtlog.md [hunk 1]
```diff
@@ -39,6 +40,12 @@ Here is a minimum configuration example.
 
 Note that some Windows Event Log channels \(like `Security`\) requires an admin privilege for reading. In this case, you need to run fluent-bit as an administrator.
 
+#### Query Languages for Event_Query Parameter
+
+`Event_Query` parameter can handle XML query for filtering Windows EventLog when collecting them.
```
Comment (patrick-stephens) [inline pipeline/inputs/windows-event-log-winevtlog.md:45]:
```suggestion
`Event_Query` parameter can be used to generate XML query for filtering Windows EventLog during collection.
```

File: pipeline/inputs/windows-event-log-winevtlog.md [hunk 2]
```diff
@@ -39,6 +40,12 @@ Here is a minimum configuration example.
...
+This parameter is able to handle both of [XPath](https://developer.mozilla.org/en-US/docs/Web/XPath) and XML Query queries.
```
Comment (patrick-stephens) [inline pipeline/inputs/windows-event-log-winevtlog.md:46]:
```suggestion
The supported query types are [XPath](https://developer.mozilla.org/en-US/docs/Web/XPath) and XML Query.
```

Link to XML Query?

File: pipeline/inputs/windows-event-log-winevtlog.md [hunk 3]
```diff
@@ -39,6 +40,12 @@ Here is a minimum configuration example.
...
+For further description, please refer [the MSDN doc](https://learn.microsoft.com/en-us/windows/win32/wes/consuming-events).
```
Comment (patrick-stephens) [inline pipeline/inputs/windows-event-log-winevtlog.md:47]:
```suggestion
For further details, please refer to [the MSDN doc](https://learn.microsoft.com/en-us/windows/win32/wes/consuming-events).
```

File: pipeline/inputs/windows-event-log-winevtlog.md (hunk 2 above)
Comment (cosmo0920) [reply pipeline/inputs/windows-event-log-winevtlog.md:46]: XML Query is having some of the specific syntax for querying Windows EventLog. However, I wasn't able to find the suitable page which describes the abstract of the syntax....

File: pipeline/inputs/windows-event-log-winevtlog.md (hunk 1 above)
Comment (patrick-stephens) [reply pipeline/inputs/windows-event-log-winevtlog.md:45]: What is an XML query though? That's probably the main thing - XML is a generic marshalling tech so is this something specific?

Comment (cosmo0920) [reply pipeline/inputs/windows-event-log-winevtlog.md:45]: I have no idea to describe simply about XML query - which can be composable and declarative format of query language for filtering Windows EventLog.

XML Query language is described here: https://learn.microsoft.com/en-us/windows/win32/wes/consuming-events
But, this is the introductory and not fundamental information of the query language.

Comment (patrick-stephens) [reply pipeline/inputs/windows-event-log-winevtlog.md:45]: Ah ok, so XML query is not some special MS thing - it just means query via XML using XPath.
```suggestion
The `Event_Query` parameter can be used to specify the XML query for filtering Windows EventLog during collection.
```

**Review by patrick-stephens — APPROVED** [review]

**Comment by cosmo0920** [comment]
Oh, we need to backport this doc PR into 2.1...

**Pull request merged**